ccf_transaction_processor/scripts/utils.py

```diff
@@ -29,11 +29,13 @@ def parse_ledger_url(config = None):
 
     if config:
         (host, port) = config["rpc-address"].split(':')
+        host = socket.gethostbyname(host)
```

Comment:
small nit... but it should be (hostname, port) = config(), then host = gethostbyname(hostname). just for clarity & good software engineering.